libvirt/resource_libvirt_domain.go

```diff
@@ -996,3 +1048,56 @@ func getDomainInterfaces(domain *libvirt.VirDomain) ([]libvirt.VirDomainInterfac
 
 	return interfaces, nil
 }
+
+func getDomainInterfacesFromNetworks(domain defDomain, meta interface{}) ([]libvirt.VirDomainInterface, error) {
+
+	log.Printf("[DEBUG] getting domain addresses from networks")
+	var virDomain = make([]libvirt.VirDomainInterface, 0)
```

Comment:
`virDomain` looks like a _domain_, but it is really a _list of interfaces_. What do you think about `ifacesLst := make([]libvirt.VirDomainInterface, 0)`?